GILua/src/main/java/org/anime_game_servers/gi_lua/script_lib/ScriptLib.kt

```diff
@@ -3728,6 +3728,1057 @@ object ScriptLib {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                showReminderByUid(this@onGroupContext, uidList, reminderId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun ShowTemplateReminder(context: LuaContextWrapper, reminderId: Int, timerInfo: IntArray): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                showTemplateReminder(this@onGroupContext, reminderId, timerInfo)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun SkipTeyvatTime(context: LuaContextWrapper, time: Int, rate: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                skipTeyvatTime(this@onGroupContext, time, rate)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun StartGadgetPlay(context: LuaContextWrapper, groupId: Int, configId: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                checkGroupIdAndConfigId(::StartGadgetPlay, groupId, configId)?.let {
+                    return@onScriptLibHandler it.getValue()
+                }
+                startGadgetPlay(this@onGroupContext, groupId, configId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun StartSceneMultiStagePlayStage(
+        context: LuaContextWrapper,
+        index: Int,
+        time: Int,
+        key: String,
+        paramTable: LuaTable
+    ): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                startSceneMultiStagePlayStage(this@onGroupContext, index, time, key, paramTable)
+                //TODO implement paramTable contains int previewStageIndex, int previewDisplayDuration
+            }
+        }
+    }
+
+    @JvmStatic
+    fun StopFishing(context: LuaContextWrapper, uid: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                stopFishing(this@onGroupContext, uid)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun StopReminder(context: LuaContextWrapper, reminderId: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                stopReminder(this@onGroupContext, reminderId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun SwitchSceneEnvAnimal(context: LuaContextWrapper, animalId: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                switchSceneEnvAnimal(this@onGroupContext, animalId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun TryRecordActivityPushTips(context: LuaContextWrapper, pushTipId: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                tryRecordActivityPushTips(this@onGroupContext, pushTipId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun UpdateStakeHomePlayRecord(context: LuaContextWrapper, uidList: IntArray): Int {
```

Comment:
probably need to be handles as lua table and then made into an IntArray, like its in AttachGalleryTeamAbilityGroup